### PR TITLE
Rs 2022 03 27

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 10/10_00_ARC10_1.txt [hunk 1]
```diff
@@ -123,7 +124,7 @@ that the time is just past 9 in the morning,
 [sha:d3b6033e9e4d7c76b6b1fe5fd7f842e4a7262f8f]{
 -- Page 7, Offset 24283.
 -- ついでに言うと、アルクェイドはブーツのままベッドに乗っている事しか分からない。
-and to top it all off, Arcueid is on my bed, still wearing her boots.
+and that to top it all off, Arcueid is on my bed, boots and all.
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_1.txt:127]: not sure about this. "boots and all" is like what I'd expect if she was some character known for her boots. I can't really put it into words, but the emphasis isn't "that crazy arcueid wearing her shoes on my bed", it's "that arcueid and her crazy boots, on my bed".

File: script/Arcueid/Day 10/10_00_ARC10_1.txt [hunk 2]
```diff
@@ -261,22 +263,28 @@ I understand the reason behind Arcueid's anger.
...
+Seeing as how the window's open,
+ that's probably the route she used to sneak her way in.
 }
 [sha:9dfcc37c7ac24b28ec23a620e5b9b9a755ecf509]{
 -- Page 19, Offset 24313.
 -- 「……そうか、約束を破ったんだから、そりゃあ、たしかに俺が悪い。けど、そんな事で<人|ひと>の<家|いえ>に侵入するのはやりすぎだぞ」
-"...I see. I was the one that broke the promise, so it's true that I'm at fault. But breaking into someone's house because they broke a promise is way worse."
+"...I see.
+ I did break my promise, so I can't deny that I'm at fault there.
+ But breaking into someone's house because they didn't keep a promise is a little excessive."
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_1.txt:287]: "a little excessive" for やりすぎだぞ is pretty downplayed.

File: script/Arcueid/Day 10/10_00_ARC10_1.txt [hunk 3]
```diff
@@ -311,98 +322,115 @@ I told myself that I would never let it end like this. That I would come here an
...
+ the sort where lashing out only causes you to sink deeper and deeper?
+ I was terrified to think about who I might become."
 }
 [sha:123f5b51a22022746926dc642f2d3df350b515f7]{
 -- Page 22, Offset 24322.
 -- 今でもそういう気分なのか、アルクェイドの赤い瞳は俺への不満・抗議・怒り・文句に満ちていた。
 // 文句 intentionally omitted from the list for redundancy.
-I wonder if she still feels that way, because Arcueid's red eyes are filled with discontent, protest, rage toward me.
+I wonder if she still feels that way,
+ because Arcueid's red eyes are brimming with a mixture of discontent, protest, and rage,
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_1.txt:343]: discontent, protest, and rage--all directly

File: script/Arcueid/Day 10/10_00_ARC10_1.txt [hunk 4]
```diff
@@ -462,52 +493,53 @@ I make that declaration while staring Arcueid directly in the face.
...
 -- Page 36, Offset 24354.
 -- シーツに足跡がついたら洗濯が大変だな、とか思わないのか」
-Didn't you even think how difficult it would be to wash footprints off the sheets?"
+Do you have any idea how difficult it is to wash footprints out of sheets?"
 }
 [sha:a1bdd547d154e0081d611e6a0d5ab8d06f549220]{
 -- Page 37, Offset 24355.
 -- 文句を言いつつベッドから体を起こす。
-I pull myself out of bed, complaining.
+I pull myself upright, complaining all the while.
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_1.txt:527]: drop "all the while" imo

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 10/10_00_ARC10_1.txt (hunk 1 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 10/10_00_ARC10_1.txt:127]: > ... all off, Arcueid hasn't even taken her boots off before getting on my bed.

?

Comment (Avuxo) [reply script/Arcueid/Day 10/10_00_ARC10_1.txt:127]: Sure, that works

**Pull request merged**